routing/routing_session.hpp

```diff
@@ -34,13 +35,34 @@ class RouteMatchingInfo;
 
 namespace routing
 {
-struct SpeedCameraRestriction
+struct SpeedCameraOnRoute
 {
-  size_t m_index;  // Index of a polyline point where camera is located.
-  uint8_t m_maxSpeedKmH;  // Maximum speed allowed by the camera.
+  SpeedCameraOnRoute() = default;
+  SpeedCameraOnRoute(double distFromBegin, uint8_t maxSpeedKmH)
+    : m_distFromBeginMeters(distFromBegin), m_maxSpeedKmH(maxSpeedKmH)
+  {}
+
+  // Needs for calculate distance for slow down speed before speed camera.
```

Comment:
Used for calculating distance needed to slow down before a speed camera.

nit: see https://en.wikipedia.org/wiki/Braking_distance

Reply:
done
make time for decision to 2 seconds